src/logic/calculate.js

```diff
@@ -0,0 +1,21 @@
+/* eslint no-unused-vars: 0 */
```

Comment:
- Fix that error. It's easy and straightforward.

Reply:
Fixed